coding-concepts.js

```diff
@@ -9,46 +9,46 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 5
+// b) Verify and explain: .push() adds the element to the end and returns the  new value of the array
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 9 
+// b) Verify and explain: It will log the length of the string, starting from 0 index.
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: l
+// b) Verify and explain: Now i see what i did wrong. Previous question was a string, so count started at 1. Since we used bracket notation here, count started from 0.
```

Comment:
Yes .length counts each character, so it starts at 1. When we are talking about index, index always starts from 0.